pages/api/auth/[...nextauth].js

```diff
@@ -6,6 +6,8 @@ import Cookies from 'cookies'
 
 import Models from '../../../models'
 
+const DATABASE_URL = `postgres://${process.env.PGUSER}:${process.env.PGPASSWORD}@${process.env.PGHOST}/${process.env.PGDATABASE}`
```

Comment:
Agreed that it is better to construct the URL out of previously established env variables rather than create a new one. The change reduces redundancy.

Reply:
Great! 👍